core-domain/src/test/java/org/uniprot/core/uniprotkb/comment/CommentTypeTest.java

```diff
@@ -56,7 +56,7 @@ void toXmlDisplayName_displayName_areNotSame() {
     }
 
     @Test
-    void canGetDefaultEvidenceCode() {
-         assertEquals(EvidenceCode.ECO_0000269, CommentType.PHARMACEUTICAL.getDefaultEvidenceCode());
+    void canGetIsAddExperimental() {
+        assertTrue(CommentType.PHARMACEUTICAL.isAddExperimental());
     }
 }
```

Comment:
addExperimental == false case is not covered.
I think, it would be good to have two paramterised test to cover true and false commenttype and featuretype.

Reply:
Please merge once you add the test